pubspec.yaml

```diff
@@ -1,5 +1,7 @@
 name: dartlang_project_templates
-version: 0.1.0
+# If you plan to publish this package, set the version field
```

Comment:
Perhaps not needed, but to be super explicit, perhaps change the text to:
`If intended for publishing, replace 'publish_to: none' with 'version:0.0.1' `

?

Reply:
👍

Reply:
FYI: the suggest replacement feature is SUPER NICE! 😄